docs/vuex_actions.md

```diff
@@ -103,8 +103,7 @@ remove head components (from page or layout)
 <a name="module_component-data.addCreatedComponentsToPageArea"></a>
 
 ### component-data.addCreatedComponentsToPageArea(store, newComponents, currentURI, path, replace, number, array, boolean) ⇒ <code>Promise</code>
-add components to a page area when they have already been created
-when the page is being reverted to the published version for instance
+add components to a page area
```

Comment:
Why did we reduced this sentence?

Reply:
Nice catch. This was a mistake
Thanks!!!